### DISCUSSION TITLE
Wow Ammzing ...

### DISCUSSION THREAD

**Discussion by verydarksun** [tillt/KompleteSynthesia#19] [General]
my only question is what is ' multiDevice.xml ' i can't find it on me system.

but 10 out 10 .. for the app.

**Comment by tillt** [comment]
For being able to configure Synthesia, KompleteSynthesia needs to manipulate a file.  `multiDevice.xml` is a configuration file belonging to Synthesia. That configuration file is typically located in the folder `~/Library/Application Support/Synthesia`.

**Comment by mickdab** [comment]
@verydarksun Hi. Did you resolve this issue. I reinstalled Synthesia and cannot find this Library. I use Sonoma.

**Comment by tillt** [comment]
@mickdab use `Finder.app`, press the key-combination `Shift-Command-G`, then enter `~/Library/Application Support/Synthesia`. It should reveal a folder that contains the file `multiDevice.xml`. Does it? If not, we need to find out why that file is not there or that folder does not exist at all.

**Reply by mickdab** [reply]
Found it. It was in user Library which was hidden. Previously i checked in general Libraries
